definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -773,6 +773,19 @@
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "05/08/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "changeServiceMethod",
+    "Name": "Change service method",
+    "Description": "Change the solicitors service method",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AosOverdue;AosAwaiting",
+    "PostConditionState": "Issued",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
you don't need these fields here.

Please remove
```
"RetriesTimeoutURLAboutToSubmitEvent": "120,120",
"RetriesTimeoutURLSubmittedEvent": "120,120",
```

Reply:
Addressed in 4b25a21f5bf186026c7933e86f859a8449132a4f